config/main.py

```diff
@@ -1515,6 +1515,18 @@ def reload(db, filename, yes, load_sysinfo, no_service_restart, force, file_form
             click.echo("Input {} config file(s) separated by comma for multiple files ".format(num_cfg_file))
             return
 
+    #Validate config_db.json file
+    if file_format == 'config_db':
+        file = DEFAULT_CONFIG_DB_FILE
```

Comment:
>file [](http://example.com/codeflow?start=8&length=4)

`file` could be `DEFAULT_CONFIG_DB_FILE` or `filename`. 
`cfg_files` could be split from `filename` or `/etc/sonic/config_db{}.json`.

Please check nearby code.